src/java.management/share/classes/javax/management/loading/MLet.java

```diff
@@ -1144,16 +1145,7 @@ private synchronized String loadLibraryAsResource(String libname) {
                                                       libname + ".", null)
                                       .toFile();
                      file.deleteOnExit();
-                     FileOutputStream fileOutput = new FileOutputStream(file);
-                     try {
-                         byte[] buf = new byte[4096];
-                         int n;
-                         while ((n = is.read(buf)) >= 0) {
-                            fileOutput.write(buf, 0, n);
-                         }
-                     } finally {
-                         fileOutput.close();
-                     }
+                     Files.copy(is, file.toPath(), StandardCopyOption.REPLACE_EXISTING);
                      if (file.exists()) {
```

Comment:
You might have missed the comment from a previous iteration. The files.exists() check goes away when Files.copy succeeds.

Reply:
You are right. Fixed.